intellij-plugin/src/main/kotlin/io/github/ethersync/protocol/EthersyncEditorProtocol.kt

```diff
@@ -0,0 +1,12 @@
+package io.github.ethersync.protocol
+
+import com.google.gson.JsonObject
+import org.eclipse.lsp4j.jsonrpc.services.JsonNotification
+import org.eclipse.lsp4j.jsonrpc.services.JsonRequest
+import java.util.concurrent.CompletableFuture
+
+interface EthersyncEditorProtocol {
+   @JsonNotification("cursor")
+   fun cursor(cursorEvent: CursorEvent)
```

Comment:
@blinry, is it possible to be notified about the own user id? This would make it a bit easier to filter out, the own cursor events.